src.R

```diff
@@ -25,7 +25,56 @@ library(tidyverse)
 
 # Geolocated Performers
 dat <- read_feather(here::here("data", "geolocated_performers_dt.feather")) %>% 
-  mutate(birth_year = as.numeric(gsub("-.*", "", birthDate)))
+  mutate(birth_year = as.numeric(gsub("-.*", "", birthDate))) %>% 
+  mutate(
```

Comment:
Not that we care about performance, but multiple mutates is more costly than a big mutate, since this is all the same type of string formatting it looks like it could all be collapses into a single `mutate(case_when(..))`

Reply:
Got it. Thanks @nathancday. Agreed. Are you still up for meeting at 5:00?

Reply:
@nathancday, I wanted to double-check with you before I start addressing my assigned issues. Is it OK to merge these commits into master as is? Or do I need to close this commit and rebase onto master on my local branch first? When you have a chance, please let me know. Also, could you please let me know your RStudio Community handle? I couldn't find you in a search and we need it for the submission (due 5:00 p.m. on Friday). Thanks!